font/fonts.css

```diff
@@ -0,0 +1,21 @@
+@font-face {
+    font-family: 'Gotham Pro';
+    src:
+      local('Gotham Pro Bold'),
+      local('GothamPro-Bold'),
+      url('GothamPro-Bold.woff2') format('woff2'),
+      url('GothamPro-Bold.woff') format('woff');
+    font-weight: bold;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Mandri_Firsttask/pullRequest?prid=5484013)